capella/versions/6.0.0/dropins.yml

```diff
@@ -38,14 +49,3 @@ dropins:
     eclipseRepository: 'jar:https://download.eclipse.org/capella/addons/subsystemtransition/updates/releases/1.6.1/SubsystemTransition-updateSite-1.6.1.202302141252.zip!/'
     installIU:
       - 'org.polarsys.capella.transition.system2subsystem.feature.feature.group'
-  TextualEditor:
-    github: 'https://github.com/eclipse/capella-textual-editor'
-    type: updateSite
-    eclipseRepository: 'jar:https://download.eclipse.org/capella/addons/textualeditor/updates/releases/0.11.0/TextualEditor-updateSite-0.11.0.202208161352.zip!/'
-    installIU:
-      - 'org.polarsys.capella.scenario.editor.feature.feature.group'
```

Comment:
I've noticed that TextualEditor was dropped with the reason that we are not aware of any active users with the dropin. Is there any additional reason for it? The `dropins.yml` should represent a full list of available dropins. The user can decide which dropins to install from the list. It does not represent the dropins which are actually installed in the images. See also: https://dsd-dbs.github.io/capella-dockerimages/capella/base/#install-dropins
 
The removal of the dropin from the list of supported dropins is a breaking change and would require a new major release.